### DISCUSSION TITLE
Documentation website for graphql-ws

### DISCUSSION THREAD

**Discussion by 97amarnathk** [enisdenjo/graphql-ws#213] [Ideas]
Would be really nice to have a website for this framework with documents and sample recipes categorized.

Some example doc sites I find really helpful:
* https://typegraphql.com/
* https://typegoose.github.io/typegoose/

Or another option would be to use [GitBooks](https://www.gitbook.com/)

**Comment by enisdenjo** [comment]
I dont think it is really necessary. It is just one more thing to maintain, coordinate and update.

The library is so simple that 90% of the time you find what you need in the [`README.md`](https://github.com/enisdenjo/graphql-ws/blob/master/README.md). If you're into advanced stuff or need API details outside your code editor, the well structured and automatically generated [`docs/`](https://github.com/enisdenjo/graphql-ws/tree/master/docs) folder is the way to go.

Additionally, I like keeping things in one place - everything from issues, discussions, changelogs, to releases, documentation, CI, environments, can be found in this repo without having to move anywhere.
